packages/pwa/app/utils/routes-utils.test.js

```diff
@@ -33,12 +43,191 @@ describe('configureRoutes', function() {
+    '/RefArch/en-HK/category/:categoryId',
+    '/RefArch/en-GB/category/:categoryId',
+    '/RefArch/uk/category/:categoryId',
+    '/us/en-US/category/:categoryId',
+    '/us/en/category/:categoryId',
+    '/us/fr-FR/category/:categoryId',
+    '/us/fr/category/:categoryId',
+    '/us/it-IT/category/:categoryId',
+    '/us/it/category/:categoryId',
+    '/us/en-HK/category/:categoryId',
+    '/us/en-GB/category/:categoryId',
+    '/us/uk/category/:categoryId',
+    '/eu/en-US/category/:categoryId',
+    '/eu/en/category/:categoryId',
+    '/eu/fr-FR/category/:categoryId',
+    '/eu/fr/category/:categoryId',
+    '/eu/it-IT/category/:categoryId',
+    '/eu/it/category/:categoryId',
+    '/eu/en-HK/category/:categoryId',
+    '/eu/en-GB/category/:categoryId',
+    '/eu/uk/category/:categoryId',
+    '/RefArchGlobal/en-US/category/:categoryId',
+    '/RefArchGlobal/en/category/:categoryId',
+    '/RefArchGlobal/fr-FR/category/:categoryId',
+    '/RefArchGlobal/fr/category/:categoryId',
+    '/RefArchGlobal/it-IT/category/:categoryId',
+    '/RefArchGlobal/it/category/:categoryId',
+    '/RefArchGlobal/en-HK/category/:categoryId',
+    '/RefArchGlobal/en-GB/category/:categoryId',
+    '/RefArchGlobal/uk/category/:categoryId',
+    '/global/en-US/category/:categoryId',
+    '/global/en/category/:categoryId',
+    '/global/fr-FR/category/:categoryId',
+    '/global/fr/category/:categoryId',
+    '/global/it-IT/category/:categoryId',
+    '/global/it/category/:categoryId',
+    '/global/en-HK/category/:categoryId',
+    '/global/en-GB/category/:categoryId',
+    '/global/uk/category/:categoryId',
+    '/RefArch/category/:categoryId',
+    '/us/category/:categoryId',
+    '/eu/category/:categoryId',
+    '/RefArchGlobal/category/:categoryId',
+    '/global/category/:categoryId',
+    '/en-US/category/:categoryId',
+    '/en/category/:categoryId',
+    '/fr-FR/category/:categoryId',
+    '/fr/category/:categoryId',
+    '/it-IT/category/:categoryId',
+    '/it/category/:categoryId',
+    '/en-HK/category/:categoryId',
+    '/en-GB/category/:categoryId',
+    '/uk/category/:categoryId',
+    '/category/:categoryId'
+]
+const mockConfig = {
+    app: {
+        defaultSite: 'RefArch',
+        url: {
+            site: 'none',
+            locale: 'none'
+        },
+        hosts: [
+            {
+                domain: 'www.domain-1.com',
+                defaultSite: 'RefArch',
+                sites: [
+                    {
+                        id: 'RefArch',
+                        alias: 'us',
+                        l10n: {
+                            supportedCurrencies: ['USD'],
+                            defaultCurrency: 'USD',
+                            defaultLocale: 'en-US',
+                            supportedLocales: [
+                                {
+                                    id: 'en-US',
+                                    alias: 'en',
+                                    preferredCurrency: 'USD'
```

Comment:
`defaultCurrency`: Matches the default currency concept set in Business Manager (Merchant Tools> Site Preferences> Currencies > Default).
![Screen Shot 2022-01-25 at 5 07 51 PM](https://user-images.githubusercontent.com/6024102/151086481-1cd2a4b3-4979-4e6d-bda6-c3a836d9f767.png)

`preferredCurrency`: It's a PWA concept. It's the currency that we'd prefer to use for the selected locale.
Background: We wanted to simplify the UI. Instead of showing two dropdowns, one for the locale and another for the currency, we show only the locale dropdown and for each locale we pre-assign a currency (preferredCurrency).
![Screen Shot 2022-01-25 at 5 15 40 PM](https://user-images.githubusercontent.com/6024102/151086714-795fd175-3738-4d94-930b-b43a71ab0ca0.png)

Reply:
This is interesting. So it seems we are basing our configurations for the UI that we have for our demo site, and I'm thinking if that's going to work for all apps.  From what I understand, If developers want to separate the drop-down for currency and locale, I believe they would still be able to do so, but just won't use the preferred currency. And the available currencies in the drop-down will depend on which locale is selected. is that right?